x-pack/plugins/enterprise_search/public/applications/app_search/components/data_panel/data_panel.tsx

```diff
@@ -45,29 +49,36 @@ export const DataPanel: React.FC<Props> = ({
   });
 
   return (
-    <EuiPanel {...props} color={filled ? 'subdued' : 'plain'} className={classes} hasShadow={false}>
+    <EuiPanel
+      {...props}
+      color={filled ? 'subdued' : 'plain'}
+      className={classes}
+      hasShadow={false}
+      aria-busy={isLoading}
+    >
       <EuiFlexGroup justifyContent="spaceBetween" alignItems="center" responsive={false}>
         <EuiFlexItem>
-          <EuiFlexGroup>
+          <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
             {iconType && (
-              <EuiFlexItem>
+              <EuiFlexItem grow={false}>
```

Comment:
This fixes the icon position. It should now look like so:

<img width="612" alt="" src="https://user-images.githubusercontent.com/549407/110559549-4521ff00-80f9-11eb-8b57-d1b5a64336aa.png">